src/main/java/com/minecolonies/coremod/entity/ai/citizen/lumberjack/Tree.java

```diff
@@ -168,7 +168,10 @@ public static boolean checkTree(@NotNull final IBlockAccess world, final BlockPo
      */
     @NotNull
     private static Tuple<BlockPos, BlockPos> getBottomAndTopLog(
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) The Cyclomatic Complexity of this method "getBottomAndTopLog" is 13 which is greater than 10 authorized. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AMethodCyclomaticComplexity)